src/modules/module1.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-plusplus */
+const home = document.getElementById('home_link');
+
+const LOCAL_STORAGE_LIST_KEY = 'card.list';
+const cards = JSON.parse(localStorage.getItem(LOCAL_STORAGE_LIST_KEY)) || [];
+
+const clearElement = ((element) => {
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+});
```

Comment:
```suggestion
element.innerHTML = '';
```
- [ ] [OPTIONAL] This also would clear the element content in one line :+1: